docroot/modules/custom/yukon_moderation/src/Controller/NodeAllRevisionsController.php

```diff
@@ -0,0 +1,263 @@
+          if ($moderation_info->isModeratedEntity($translated_revision)) {
+            $workflow = $moderation_info->getWorkflowForEntity($translated_revision);
+            if ($workflow) {
+              $current_state = $translated_revision->get('moderation_state')->value;
+              if ($current_state) {
+                $workflow_states = $workflow->getTypePlugin()->getStates();
+                if (isset($workflow_states[$current_state])) {
+                  $status = $workflow_states[$current_state]->label();
+                }
+              }
+            }
+          }
+        }
+
+        // If content moderation didn't work, try basic published/unpublished.
+        if ($status == $this->t('Draft')) {
+          $status = $translated_revision->isPublished() ? $this->t('Published') : $this->t('Draft');
+        }
+
+        // We treat also the latest translation-affecting revision as current
+        // revision, if it was the default revision, since its values for the
+        // current language will be the same of the current default revision in
+        // this case.
+        $is_current_revision = $vid == $default_revision || (!$current_revision_displayed && $translated_revision->wasDefaultRevision());
+
+        if (!$is_current_revision) {
+          $link = Link::fromTextAndUrl($date, new Url('entity.node.revision', [
+            'node' => $node->id(),
+            'node_revision' => $vid,
+          ], ['language' => $language]));
+        }
+        else {
+          $link = $node->toLink($date, 'canonical', ['language' => $language]);
+          $current_revision_displayed = TRUE;
+        }
+
+        $link_renderable = $link->toRenderable();
+        $username_renderable = $username;
+
+        $row = [];
+        $column = [
+          'data' => [
+            '#type' => 'inline_template',
+            '#template' => '{% trans %}{{ date }} by {{ username }}{% endtrans %}<p class="revision-status">({{ language }} - {{ status }})</p>{% if message %}<p class="revision-log">{{ message }}</p>{% endif %}',
+            '#context' => [
+              'date' => Markup::create($this->renderer->render($link_renderable)),
+              'username' => Markup::create($this->renderer->render($username_renderable)),
+              'message' => Markup::create($translated_revision->getRevisionLogMessage()),
+              'language' => $language->getName(),
+              'status' => $status,
+            ],
+          ],
+        ];
+
+        $this->renderer->addCacheableDependency($column['data'], $translated_revision);
+
+        $row[] = $column;
+
+        if ($is_current_revision) {
+          $row[] = [
+            'data' => [
+              '#prefix' => '<em>',
+              '#markup' => $this->t('Current revision'),
+              '#suffix' => '</em>',
+            ],
+          ];
+
+          $row['#attributes'] = [
+            'class' => ['revision-current'],
+          ];
```

Comment:
I think this is causing an extra column to be added.

<img width="1022" height="432" alt="Screenshot 2025-07-25 at 3 49 03 PM" src="https://github.com/user-attachments/assets/e1602ef1-6c63-4cba-8e9e-5af9920cdd29" />